### DISCUSSION TITLE
Welcome to WakayakosenRoboconClub Discussions!

### DISCUSSION THREAD

**Discussion by gpgpc0** [gpgpc0/Wakayama-kosen-RoboconClub#1] [Announcements]
[JP]
## 👋 ようこそ！
  ディスカッションは、私たちのコミュニティの他のメンバーとつながるための場所として使用しています。ぜひご利用ください：
  * 疑問に思っていることを質問してください。
  * アイデアを共有する。
  * 他のコミュニティメンバーと交流する。
  * 他の人を歓迎し、オープンマインドであること。このコミュニティは私たちが
  一緒に作り上げるコミュニティであることを忘れないでください。

  まずは、自己紹介とこのコミュニティで何をするのかについて教えてください。
[EN]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by T-Kd** [comment]
よろしくお願いします
